docs/database/Status_6_Performance_Issue_Quick_Ref.md

```
@@ -0,0 +1,236 @@
+## Verification After Fix
+
+Run Status 6 query again and check:
+```sql
+EXPLAIN (ANALYZE, BUFFERS)
+SELECT DISTINCT ...
+WHERE a2Events."Status" = 6
+LIMIT 100;
+```
+
+**Expected Results**:
+| Metric | Target | Your Result |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)